test/e2e/pods/pods.go

```diff
@@ -50,6 +59,25 @@ func ExecCommand(cs *testclient.ClientSet, pod *corev1.Pod, command ...string) (
 	return buf.String(), errbuf.String(), nil
 }
 
+func ExecCommandWithRetry(client *testclient.ClientSet, sourcePod *corev1.Pod, timeout time.Duration, command ...string) (string, string, error) {
+	var err error
+	var stdOut, stdErr string
+
+	for start := time.Now(); time.Since(start) < timeout; time.Sleep(2 * time.Second) {
+		stdOut, stdErr, err = ExecCommand(client, sourcePod, command...)
+		if err != nil {
+			log.Printf("got err: %v, retry until timeout", err)
+			continue
+		}
+		if strings.TrimSpace(stdErr) != "" {
+			log.Printf("got non-empty stderr %q, retry until timeout", stdErr)
```

Comment:
does those logs fill generate lots of noise during running e2e ?

Reply:
nope - never seen them output.